.vscode/settings.json

```diff
@@ -6,7 +6,6 @@
 	"cssvar.extensions": ["js", "css", "html", "jsx", "tsx", "svelte"],
 	"python.analysis.extraPaths": ["./gradio/themes/utils"],
 	"prettier.useTabs": true,
```

Comment:
Could you remove this as well? The prettier config should be the source of truth.